backstory/src/main/webapp/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="script.js"></script>
+  </head>
+
+  <body class="vertical-flex-container" id="perspective-page">
+    <div class="content vertical-flex-container" id="perspective-content">
+      <h1 class="header blue-header">Backstory Demo</h1>
+
+      <div class="vertical-flex-container" id="demo-divs">
+        <a href="visionDemo.html"
+          ><div class="demo-div" id="vision-demo-div">
+            <h3>Cloud Vision Demo</h3>
+          </div></a
+        >
```

Comment:
I think it would be clearer to close the tag on the same line where possible, or is there guidance otherwise?

```
<a href="visionDemo.html">
  <div class="demo-div" id="vision-demo-div">
    <h3>Cloud Vision Demo</h3>
  </div>
</a>
```

Reply:
No there's not guidance otherwise. Sometimes ```make pretty``` just does that, and since I usually run out before uploading, it's hard to catch. Thanks for pointing it out. I'll fix it.

Reply:
Done